src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmounts.jsx

```diff
@@ -7,21 +7,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import ChartError from 'components/search/visualizations/ChartError';
-import AggregatedAwardAmountsInfo from 'components/awardv2/visualizations/amounts/AggregatedAwardAmountsInfo';
```

Comment:
Seeing import paths like this vs whats on `line 12` has been confusing to me. Would like to understand how these kinds of imports even worked in the past?

Reply:
Resolved by the Webpack setting [here](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/webpack/webpack.common.js#L9)